remove-vowels/mutation_1.py

```diff
@@ -0,0 +1,11 @@
+def remove_vowels(strng: str) -> str:
+    """
+    Bug: Implementation that keeps vowels
```

Comment:
This implementation would not "keep the vowels" whatsoever, instead, it prints nothing. Please change the explanation of the bug pookie

Reply:
thx